src/components/Search/SearchFiltersChatsSelector.tsx

```diff
@@ -0,0 +1,214 @@
+import React, {useCallback, useEffect, useMemo, useState} from 'react';
+import {useOnyx} from 'react-native-onyx';
+import Button from '@components/Button';
+import {usePersonalDetails} from '@components/OnyxProvider';
+import {useOptionsList} from '@components/OptionListContextProvider';
+import SelectionList from '@components/SelectionList';
+import InviteMemberListItem from '@components/SelectionList/InviteMemberListItem';
+import useDebouncedState from '@hooks/useDebouncedState';
+import useLocalize from '@hooks/useLocalize';
+import useScreenWrapperTranstionStatus from '@hooks/useScreenWrapperTransitionStatus';
+import * as DeviceCapabilities from '@libs/DeviceCapabilities';
+import * as OptionsListUtils from '@libs/OptionsListUtils';
+import type {Option} from '@libs/OptionsListUtils';
+import type {OptionData} from '@libs/ReportUtils';
+import * as ReportUtils from '@libs/ReportUtils';
+import Navigation from '@navigation/Navigation';
+import * as Report from '@userActions/Report';
+import CONST from '@src/CONST';
+import ONYXKEYS from '@src/ONYXKEYS';
+import ROUTES from '@src/ROUTES';
+
+const defaultListOptions = {
+    recentReports: [],
+    personalDetails: [],
+    userToInvite: null,
+    currentUserOption: null,
+    categoryOptions: [],
+    tagOptions: [],
+    taxRatesOptions: [],
+    headerMessage: '',
+};
+
+function getSelectedOptionData(option: Option): OptionData {
+    return {...option, selected: true, reportID: option.reportID ?? '-1'};
+}
+
+type SearchFiltersParticipantsSelectorProps = {
+    initialAccountIDs: string[];
+    onFiltersUpdate: (accountIDs: string[]) => void;
+    isScreenTransitionEnd: boolean;
+};
+
+function SearchFiltersChatsSelector({initialAccountIDs, onFiltersUpdate, isScreenTransitionEnd}: SearchFiltersParticipantsSelectorProps) {
+    const {translate} = useLocalize();
+    const personalDetails = usePersonalDetails();
+    const {didScreenTransitionEnd} = useScreenWrapperTranstionStatus();
+    const {options, areOptionsInitialized} = useOptionsList({
+        shouldInitialize: didScreenTransitionEnd,
+    });
+
+    const [isSearchingForReports] = useOnyx(ONYXKEYS.IS_SEARCHING_FOR_REPORTS, {initWithStoredValues: false});
+    const [selectedOptions, setSelectedOptions] = useState<OptionData[]>([]);
+    const [searchTerm, debouncedSearchTerm, setSearchTerm] = useDebouncedState('');
+    const cleanSearchTerm = useMemo(() => searchTerm.trim().toLowerCase(), [searchTerm]);
+
+    const defaultOptions = useMemo(() => {
+        if (!areOptionsInitialized || !isScreenTransitionEnd) {
+            return defaultListOptions;
+        }
+        const optionList = OptionsListUtils.getSearchOptions(options, '');
+        const header = OptionsListUtils.getHeaderMessage(optionList.recentReports.length + optionList.personalDetails.length !== 0, !!optionList.userToInvite, '');
+        return {...optionList, headerMessage: header};
```

Comment:
I believe this `headerMessage` here might be not needed. Do you know if it affects anything?
If not remove it and return `optionList` directly.

Reply:
Good catch, the headerMessage is used here but in opposite to `chat-finder`, it is set together with `sections`, this part is useless then 👍